src/foundry/client-esm/applications/api/document-sheet.d.mts

```diff
@@ -42,7 +42,8 @@ declare class DocumentSheetV2<
   Document extends foundry.abstract.Document<any, any, any>,
   Configuration extends DocumentSheetV2.Configuration<Document> = DocumentSheetV2.Configuration<Document>,
   RenderOptions extends DocumentSheetV2.RenderOptions = DocumentSheetV2.RenderOptions,
-> extends ApplicationV2<Configuration, RenderOptions> {
+  RenderContext extends Record<string, unknown> = Record<string, never>,
```

Comment:
I decided that every subclass should also allow this to be overridden. This is somewhat based upon gut instinct; someone may extend `DocumentSheetV2` (or other subclasses) and want to also override `_onRender`.

I could not determine a way in which this would be problematic.

Reply:
_onRender is the new activateListeners so yeah definitely a good call